starlette/middleware/base.py

```diff
@@ -47,11 +47,16 @@ async def coro() -> None:
             assert message["type"] == "http.response.start"
 
             async def body_stream() -> typing.AsyncGenerator[bytes, None]:
+                nonlocal app_exc
```

Comment:
This shouldn't be necessary, right?

Reply:
You are right, there is no assignment to `app_exc`. Removed.